Tests/Validator/UniqueUrlValidatorTest.php

```diff
@@ -99,6 +102,7 @@ public function testValidateWithRootUrlAndNoParent()
 
     public function testValidateWithPageDynamic()
     {
+        $this->skipInPHP55();
```

Comment:
I few months ago, I got the same error. 

It's kind of a PHPUnit bug, when you declare a class alias in the `use` section which already exists as a real class name. E.g. your class `Item` is in the `AppBundle\Entity` namespace and you are importing the class `use FooBundle\Bar\Baz;` But a class `AppBundle\Entity\Baz` exists.

Sadly, I can't find the code which fixes the problem, but i think defining a class alias like `use FooBundle\Bar\Baz as TestBaz;` should fix the problem.